svc-lighthouse-api/src/main/resources/application-nonprod.properties

```diff
@@ -0,0 +1,13 @@
+# Contains shared properties for non-prod environments (local, dev, qa, sandbox)
+# This file is used when a Spring Group Profile is set up
+# in application.properties (under spring.profiles.group) to use `nonprod`
+# See https://github.com/department-of-veterans-affairs/abd-vro/wiki/Configuration-settings#vros-use-of-spring-profiles
+# application.properties is loaded, then this file is loaded
+
+# server.port=8091
+# server.max-http-header-size=48000
```

Comment:
These lines were in `application-local.properties` -- not sure why. Can they be removed?